manager/src/main/java/com/redhat/service/smartevents/manager/workers/resources/ConnectorWorker.java

```diff
@@ -48,6 +49,21 @@ protected PanacheRepositoryBase<ConnectorEntity, String> getDao() {
     }
 
     @Override
+    protected String getId(Work work) {
```

Comment:
@manstis I don't know if this is actually ever called. It looks like `ProcessorWorker` calls the create/delete dependencies method directly on `ConnectorWorker`?